hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneKeyLocation.java

```diff
@@ -39,16 +39,20 @@
    * Offset of this key.
    */
   private final long offset;
-
+  /**
+   * KeyOffset of this key.
+   */
+  private final long KeyOffset;
```

Comment:
```suggestion
  private final long keyOffset;
```
use lower camel case: KeyOffset -> keyOffset